src/contexts/configuration/ConfigurationContext.tsx

```diff
@@ -27,6 +28,7 @@ export const initialState = {
   APPLICATION_NAME: "",
   GooglePlayLink: "",
   IOSStoreLink: "",
+  RepoLogo: "https://cdn.afterdawn.fi/v3/news/original/github-logo.png",
```

Comment:
@sahandfarahani Isn't this incorrect?

Reply:
@ralph-dev Magic string?

Reply:
@sahandfarahani Why does the context have a default value? instead of grabbing it from the REDAME

Reply:
@ralph-dev wouldn't user specify in config if logo is wanted, or else default?

Reply:
@ralph-dev @Manpreet-Bhatti I'm not sure what is the strategy here. I only explained Manpreet how does our context system works and how can we add a key to it and how to use it. I thought maybe you had planned for an initial value for this subject.

Reply:
@manpreet this is incorrect. 	you cannot provide a default value here, it must be in readme config this needs to be empty so the system knows if a URL has been specified by the repo config file or not

Reply:
Yes but Default is no logo, not github's logo.

Reply:
Apologies If i said something else but if no logo key was provided by their config file don't auto give them a logo

Reply:
@ralph-dev all good, should be fixed now hopefully